src/constants/settings.ts

```diff
@@ -27,12 +28,14 @@ export const networkConfig: NetworkConfig = {
     network: StellarSdk.Networks.TESTNET,
     stellarExpertTxUrl: "https://stellar.expert/explorer/testnet/tx/",
     stellarExpertAccountUrl: "https://stellar.expert/explorer/testnet/account/",
+    stellarExpertAssetUrl: "https://stellar.expert/explorer/testnet/asset/",
   },
   public: {
     url: "https://horizon.stellar.org",
     network: StellarSdk.Networks.PUBLIC,
     stellarExpertTxUrl: "https://stellar.expert/explorer/public/tx/",
     stellarExpertAccountUrl: "https://stellar.expert/explorer/public/account/",
+    stellarExpertAssetUrl: "https://stellar.expert/explorer/public/asset/",
   },
```

Comment:
should we set something like `const STELLAR_EXPERT_URL = https://stellar.expert/explorer/`